frontend/src/hooks/index.ts

```diff
@@ -29,6 +29,9 @@ const GEMSContract = new ethers.Contract(GEMS_CONTRACT_ADDRESS, GEMSContractInte
 const LOOTContract = new ethers.Contract(LOOT_CONTRACT_ADDRESS, LOOTContractInterface);
 const FAUCETContract = new ethers.Contract(FAUCET_CONTRACT_ADDRESS, FAUCETContractInterface);
 
+//! don't forget to update
+const options = { chainId: ChainId.Goerli };
```

Comment:
Why do we need to add this now?

Reply:
This is needed for the read operations to be valid when the user is not connected / connected to the wrong network